third_party/2and3/click/decorators.pyi

```diff
@@ -47,7 +47,7 @@ def command(
 
 def group(
     name: Optional[str] = ...,
-    cls: type = Group,
+    cls: Type[Command] = ...,
```

Comment:
The argument defaults to Group, but any Command will work: https://github.com/pallets/click/blob/master/click/decorators.py#L121. Group is a subclass of Command.